### PR TITLE
mikeSie adds Boston, MA

### LINKED ISSUE #44: mikeSie follows GitHub Flow
# Boston, MA

- [x] create a **branch**
- [x] add some **commits**
- [x] open a *pull request*
- [ ] collaborate
- [ ] `merge` with 👍

### PR DESCRIPTION
**Pull request by mikeSie**
This Pull Request is to add Boston, MA to the list of hometowns.

This PR closes #44.

### CODE REVIEW

File: mikeSie-Boston.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Boston, MA
+
+### Restaurants
+
+- Union Oyster House
+
+### Attractions / Places to see
+ 
+- Fenway Park
```
Comment (Turnev) [inline mikeSie-Boston.md]: @mikeSie what is the Avg. ticket price

File: mikeSie-Boston.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 ### Restaurants
 
+- Union Oyster House
+
 ### Attractions / Places to see
+ 
+- Fenway Park
```
Comment (edlutes) [inline mikeSie-Boston.md]: Maybe the Sox can live up to their potential this year.

File: mikeSie-Boston.md (hunk 1 above)
Comment (mikeSie) [inline mikeSie-Boston.md]: Too expensive!  Probably over $100 on average.

File: mikeSie-Boston.md (hunk 2 above)
Comment (mikeSie) [inline mikeSie-Boston.md]: Yes, I'm optimistic!

**Pull request merged**